content/zh/releases/release.md

```diff
@@ -0,0 +1,659 @@
+  为了使特性增强成为当前版本的一部分，必须在 [KEPs][keps] 的截止日期前完成
+
+- **[异常请求][exceptions]**：
+  请求延长特定增强的截止日期的过程
+
+- **[代码冻结][code-freeze]**：
+  最终发布日期前约 4 周的时间，在此期间，仅将关键错误修复合并到发布中。
+
+- **[修剪](https://git.k8s.io/sig-release/releases/release*phases.md#pruning)**：从发布里程碑中删除增强功能的过程，如果它没有完全实现或被认为不稳定。
+
+<!-- 
+- *release milestone*: semantic version string or
+  [GitHub milestone](https://help.github.com/en/github/managing-your-work-on-github/associating-milestones-with-issues-and-pull-requests)
+  referring to a release MAJOR.MINOR `vX.Y` version.
+
+  See also
+  [release versioning](/contributors/design-proposals/release/versioning.md).
+
+- *release branch*: Git branch `release-X.Y` created for the `vX.Y` milestone.
+
+  Created at the time of the `vX.Y-rc.0` release and maintained after the
+  release for approximately 12 months with `vX.Y.Z` patch releases.
+
+  Note: releases 1.19 and newer receive 1 year of patch release support, and
+  releases 1.18 and earlier received 9 months of patch release support.
+-->
+- **发布里程碑**：语义版本字符串或 [GitHub 里程碑](https://help.github.com/en/github/managing-your-work-on-github/associating-milestones-with-issues-and-pull-requests)
+  指的是发布 主.次 `vX.Y` 版本。
+
+  另请参阅[发布版本控制](/contributors/design-proposals/release/versioning.md)。
+
+- **发布分支**：为 `vX.Y` 里程碑创建的 Git 分支 `release-X.Y`。
+
+  在 `vX.Y-rc.0` 发布时创建，并在发布后使用 `vX.Y.Z` 补丁版本，维护大约 12 个月。
+
+  注意：1.19 及更高版本获得 1 年的补丁版本支持，1.18 及更早版本获得 9 个月的补丁版本支持。
+
+<!-- 
+## The Release Cycle
+-->
+## 发布周期  {#the-release-cycle}
+
+![Image of one Kubernetes release cycle](release-cycle.jpg)
+
+<!-- 
+Kubernetes releases currently happen approximately three times per year.
+
+The release process can be thought of as having three main phases:
+
+- Enhancement Definition
+- Implementation
+- Stabilization
+-->
+Kubernetes 目前大约每年发布三次。
+
+发布过程可被认为具有三个主要阶段：
+
+- 特性增强定义
+- 实现
+- 稳定
+
+<!-- 
+But in reality, this is an open source and agile project, with feature planning
+and implementation happening at all times. Given the project scale and globally
+distributed developer base, it is critical to project velocity to not rely on a
+trailing stabilization phase and rather have continuous integration testing
+which ensures the project is always stable so that individual commits can be
+flagged as having broken something.
+-->
+但实际上，这是一个灵活的开源项目，功能规划和实施始终在进行。
+鉴于项目规模和全球分布的开发人员基础，对项目速度至关重要的是不依赖于后续的稳定阶段，
+而是进行持续集成测试，以确保项目始终稳定，以便可以将单个提交标记为有问题。
+
+<!-- 
+With ongoing feature definition through the year, some set of items will bubble
+up as targeting a given release. **[Enhancements Freeze][enhancements-freeze]**
+starts ~4 weeks into release cycle. By this point all intended feature work for
+the given release has been defined in suitable planning artifacts in
+conjunction with the Release Team's [Enhancements Lead](https://git.k8s.io/sig-release/release-team/role-handbooks/enhancements/README.md).
+-->
+随着全年功能定义的不断进行，某些项目将冒泡作为给定版本的目标。
+在发布周期约 4 周后开始**[冻结特性增强][enhancements-freeze]**。
```

Comment:
We shouldn't add space between Mandarin words, but the `**bold**` is not parsed.
```suggestion
在发布周期约 4 周后开始 **[冻结特性增强][enhancements-freeze]**。
```

Reply:
nice catch, thanks